### PR TITLE
fix(deps): ESM deps should be dev-only

### PR DESCRIPTION
**Pull request by whizzzkid**
Blocks: https://github.com/ipfs/ipfs-webui/pull/2061

In this PR:
- Updated Dependencies to be dev only.
- ESM ships with everything needed.

### CODE REVIEW

File: .aegir.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 export default {
   tsRepo: false,
+  dependencyCheck: {
+    input: [
+      'dist/**/*.js',
+    ],
+    productionInput: [
+      'dist/**/*.js',
+    ],
+  },
```
Comment (whizzzkid) [inline .aegir.js:10]: Created an issue for this: https://github.com/ipfs/aegir/issues/1104

Comment (SgtPooki) [reply .aegir.js:10]: Left a comment on https://github.com/ipfs/aegir/issues/1104#issuecomment-1301956533

**Review by SgtPooki — COMMENTED** [review]
left a comment on https://github.com/ipfs/aegir/issues/1104#issuecomment-1301956533. If those assumptions are true, then I'm fine with this PR.. lmk

**Review by SgtPooki — APPROVED** [review]

**Review by lidel — APPROVED** [review]
lgtm but we should  park this until #99 (https://github.com/protocol/.github/pull/410)  is set up, so when this PR is merged a patch release happens automatically

**Comment by whizzzkid** [comment]
@lidel absolutely.

**Comment by lidel** [comment]
Seems things are set up for automatic release: https://github.com/ipfs-shipyard/ipfs-geoip/actions/runs/3421056172/jobs/5697575441#step:6:62

Let's merge and see if it autoreleases this.

**Pull request merged**